lib/Logger.ts

```diff
@@ -55,49 +45,46 @@ type Loggers = {
 
 class Logger {
   private level: string;
-  private logDir: string;
   private context: Context;
   private logger: any;
   private instanceId: number;
 
-  private static defaultLogDir = 'logs';
-
-  constructor({ level, instanceId, logDir, context }: {level: string, instanceId?: number, logDir?: string, context: Context}) {
+  constructor({ level, file, context, instanceId }: {level: string, file: string, context: Context, instanceId?: number}) {
```

Comment:
Let's rename `file` to `filename` so we use the same name as the winston config option.